### ISSUE TITLE
FT.AGGREGATE - Can you add an option to LOAD and LOAD ALL after FILTER command

### ISSUE THREAD

**Issue by nadavtsa** [rueian/rueidis#169]
I want to be able to implement the logic in this https://github.com/RediSearch/RediSearch/issues/3229 (just scroll down to the bottom of the thread - I want to do a sort of "elem match" in a list of json objects). I basiccally want to run the following command (which I tested with redis-cli and seems to be working):
```
FT.AGGREGATE idx:user "@orders_no:ord001 @orders_id:[2 2]" LOAD 3 '$.orders[?(@.no=="ord001" && @.id==2)]' as match FILTER 'exists(@match)' LOAD *
```

From looking at your code I think all you need is to add the following receiver to /internal/cmds/gen.go:
```
func (c FtAggregateOpFilter) LoadAll() FtAggregateLoadallLoadAll {
	c.cs.s = append(c.cs.s, "LOAD", "*")
	return (FtAggregateLoadallLoadAll)(c)
}
```

**Comment by rueian** [comment]
Sure, I have made changes in https://github.com/rueian/rueidis/commit/db6fa061bfdbd4396347ae358b4e62bb29412476 so that you will able to use `LOAD` after `FILTER` in the next version.

**Comment by rueian** [comment]
Hi @nadavtsa,

https://github.com/rueian/rueidis/commit/db6fa061bfdbd4396347ae358b4e62bb29412476 is released in v0.0.93.
